github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -1048,3 +1048,28 @@ postsubmits:
                 memory: "8Gi"
               limits:
                 memory: "8Gi"
+    - name: mirror-images-from-docker-hub
+      always_run: false
+      run_if_changed: "hack/mirror-images.sh"
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      cluster: ibm-prow-jobs
+      labels:
+        preset-kubevirtci-quay-credential: "true"
+      spec:
+        containers:
+        - image: quay.io/kubevirtci/bootstrap:v20210906-994b913
```

Comment:
We should use the latest bootstrap image, it is based in f34 and ships a recent skopeo.

Reply:
Updated